CNP_analysis.py

```diff
@@ -90,36 +100,36 @@
         confoundsfile = cf_files['confoundsfile']
 
     eventsfile = os.path.join(BIDSDIR, SUBJECT, 'func',
-                              SUBJECT + "_task-" + TASK + '_events.tsv')
-    regressors = utils.create_ev_task(eventsfile, eventsdir, TASK)
+                              '%s_task-%s_events.tsv' % (SUBJECT, task_id))
+
+    regressors = utils.create_ev_task(eventsfile, eventsdir, task_id)
     EVfiles = regressors['EVfiles']
     orthogonality = regressors['orthogonal']
 
-    contrasts = utils.create_contrasts(TASK)
+    contrasts = utils.create_contrasts(task_id)
 
     # START PIPELINE
     # inputmask = Node(IdentityInterface(fields=['mask_file']), name='inputmask')
 
     if args.prep_pipeline.startswith("fsl"):
         masker = Node(ApplyWarp(
-            in_file = cf_files['bold'],
+            in_file=cf_files['bold'],
```

Comment:
function arguments should not have spaces (PEP8). Tools like autopep will fix these for you automatically.